src/plugins/enzymes/src/FindEnzymesDialog.h

```diff
@@ -134,6 +134,14 @@ class EnzymeTreeItem : public QTreeWidgetItem {
     EnzymeTreeItem(const SEnzymeData& ed);
     SEnzymeData enzyme;
     bool operator<(const QTreeWidgetItem& other) const;
+
+private:
+    enum class Ns {
```

Comment:
Document the name if values. Or use a good enum name that won't require documentation.

Reply:
OK